bookkeeper-server/src/main/java/org/apache/bookkeeper/http/DecommissionService.java

```diff
@@ -73,7 +77,7 @@ public HttpServiceResponse handle(HttpServiceRequest request) throws Exception {
                     BookieSocketAddress bookieSrc = new BookieSocketAddress(
                       bookieSrcString[0], Integer.parseInt(bookieSrcString[1]));
 
-                    Thread thread = new Thread() {
+                    executor.execute(new Runnable() {
```

Comment:
a small thing, but you don't need to do the whole runnable thing anymore in java 8. This could be
```
executor.execute(() -> {
    try {
      bka.decommissionBookie(bookieSrc);
  } catch (Exception e) {
      LOG.error("Error handling decommissionBookie: {} with exception {}", bookieSrc, e);
  }
```